execute.go

```diff
@@ -44,6 +44,7 @@ func executeWithOutputChan(outputChan chan string, env []string, workingDir stri
 	cmd := exec.Command(execPath, args...)
 	cmd.Dir = workingDir
 	cmd.Env = env
+	cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
```

Comment:
two things about this:

1. Setpgid behaves little different on linux under certain conditions and might panic. but very specific scenarios so I wouldn't be worried about that here untill we see it.
2. I closed my PR with this solution coz it was killing the parent process (setpgid was creating the process group with parent process while we need a process group only for child processes). But I am going to checkout to this pr and run some of my tests.